target/linux/lantiq/image/grx390.mk

```diff
@@ -0,0 +1,9 @@
+define Device/dlink_dwr-966
+  $(Device/NAND)
+  DEVICE_VENDOR := D-Link
+  DEVICE_MODEL := DWR-966
+  BOARD_NAME := DWR-966
```

Comment:
Is BOARD_NAME really required or can it be dropped?

Reply:
dropped